src/views/catalog/customize/components/CustomizeForm.tsx

```diff
@@ -0,0 +1,152 @@
+import * as React from 'react';
+import { useParams } from 'react-router-dom';
+
+import {
+  ProcessedTemplatesModel,
+  TemplateParameter,
+  V1Template,
+} from '@kubevirt-ui/kubevirt-api/console';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { k8sCreate } from '@openshift-console/dynamic-plugin-sdk';
+import { RedExclamationCircleIcon } from '@openshift-console/dynamic-plugin-sdk/lib/app/components/status/icons';
+import {
+  ActionGroup,
+  Button,
+  ExpandableSection,
+  Form,
+  FormGroup,
+  TextInput,
+} from '@patternfly/react-core';
+
+import { DEFAULT_NAMESPACE, VIRTUAL_MACHINE_KIND } from '../constants';
+
+type FieldGroupProps = {
+  field: TemplateParameter;
+  showError?: boolean;
+};
+
+const FieldGroup: React.FC<FieldGroupProps> = ({ field, showError }) => {
```

Comment:
nice :) 
can you extract this component to another file for better readability?

Reply:
okay